src/components/AudioTrack/Waveform.vue

```diff
@@ -343,6 +365,10 @@ export default {
 </script>
 
 <style scoped lang="css">
+.timestamp {
+  @apply absolute top-1 font-bold text-sm px-1 pointer-events-none;
+}
```

Comment:
Is `@apply` safe to use? My understanding is that the proposal for it has been abandoned?

Reply:
Doesn't seem to be abandoned from the [docs](https://tailwindcss.com/docs/extracting-components#extracting-component-classes-with-apply) where they propose it as a simpler alternative to creating a JS-framework component. Please share the link to the proposal if you have it?

Reply:
Here it says it is abandoned: https://tabatkins.github.io/specs/css-apply-rule/

But it also looks like tailwind uses it a lot so I'm confused about the status of it.

Reply:
Oh wait, is tailwind's `@apply` different from the CSS standard `@apply`? How does that work? Does tailwind take a pre-processor?

Reply:
Ah yes @sarayourfriend, the CSS draft proposal for `@apply` has been abandoned but PostCSS (which Tailwind uses) seems to work around it. We should be fine for now as this support extends into PostCSS 8 as well.

Reply:
Oh okay, `PostCSS` is so confusing to me, as it just works on regular `css` files instead of adopting a different format 🙄 Oh well, it makes sense to me to continue to use it if we're sticking with Tailwind.

Reply:
Sometimes PostCSS files have the `.pcss` extension but I've rarely seen it being used.